content/docs/glossary/multisig.mdx

```diff
@@ -143,50 +143,4 @@ Expense account setup:
 
 </CodeExample>
 
-<Alert>
 
-The next two examples involve setting the master key weight of an account to 0. Be very, very careful if you decide to do that: that key will no longer be able to sign any kind of transaction, so you are in danger of _permanently_ locking yourself out of your account. Make sure you've thought carefully about what you're doing, and that you understand the implications.
-
-</Alert>
-
-### Example 4: Company Accounts
-
```

Comment:
The company account example might still be useful. It is not encouraging the lock accounts approach so I think it can stay?

Reply:
That's a fair point.  I added a bit to the warning + the example to notify people to be careful about the order they follow when changing weights.  Thoughts @leighmcculloch ?